### PR TITLE
added basic cypress tests

### PR DESCRIPTION
**Pull request by qvalentin**

### CODE REVIEW

**Comment by qvalentin** [comment]
for installation instructions see [here](https://docs.cypress.io/guides/getting-started/installing-cypress.html)

File: .github/workflows/storybookBuild.yml [hunk 1]
```diff
@@ -2,6 +2,8 @@ name: Storybook Release
 
 on:
   pull_request:
+    paths:
+      - 'webapp_frontend/src/**'
```
Comment (qvalentin) [inline .github/workflows/storybookBuild.yml:6]: @open-schnick why does this not work, it is running everytime

**Comment by qvalentin** [comment]
Discuss: Should we use the cypress dashboard? You sadly can't host it :-1: 

https://docs.cypress.io/guides/dashboard/github-integration.html

**Comment by open-schnick** [comment]
> Discuss: Should we use the cypress dashboard? You sadly can't host it 
> 
> https://docs.cypress.io/guides/dashboard/github-integration.html

isn't it just a bot that comments on your prs whether or not the tests were green? I kinda like it ngl.

File: .github/workflows/storybookBuild.yml (hunk 1 above)
Comment (open-schnick) [reply .github/workflows/storybookBuild.yml:6]: because it uses the master yml.

Comment (open-schnick) [reply .github/workflows/storybookBuild.yml:6]: its kinda stupid. now it should work. but when you are updating a workflow it still uses the master one, but when creating one its used although its not on master.

**Comment by qvalentin** [comment]
> > Discuss: Should we use the cypress dashboard? You sadly can't host it 
> > 
> > https://docs.cypress.io/guides/dashboard/github-integration.html
> 
> isn't it just a bot that comments on your prs whether or not the tests were green? I kinda like it ngl.

Yes and also a online dashboard like Codefactor. Seems reasonable to use (angry selfhosting noises).

**Comment by open-schnick** [comment]
> > > Discuss: Should we use the cypress dashboard? You sadly can't host it
> > > https://docs.cypress.io/guides/dashboard/github-integration.html
> > 
> > 
> > isn't it just a bot that comments on your prs whether or not the tests were green? I kinda like it ngl.
> 
> Yes and also a online dashboard like Codefactor. Seems reasonable to use (angry selfhosting noises).

BD35B8F7-8DC8-48FF-A769-F81BB277BF3D

@qvalentin when something is not selfhostable

**Comment by qvalentin** [comment]
hi this comment was sent from IDEA

**Comment by qvalentin** [comment]
@FileFighter/filefighter-dev hallo anyone wanna review 

@Gimleux @open-schnick

**Review by open-schnick — APPROVED** [review]

**Pull request merged**